packages/dds/tree/src/feature-libraries/defaultFieldKinds.ts

```diff
@@ -495,19 +488,21 @@ const optionalChangeRebaser: FieldChangeRebaser<OptionalChangeset> = isolatedFie
 	invert: (
 		{ revision, change }: TaggedChange<OptionalChangeset>,
 		invertChild: NodeChangeInverter,
+		reviver: NodeReviver,
 	): OptionalChangeset => {
 		const inverse: OptionalChangeset = {};
 
 		const fieldChange = change.fieldChange;
 		if (fieldChange !== undefined) {
 			inverse.fieldChange = { wasEmpty: fieldChange.newContent === undefined };
 			if (!fieldChange.wasEmpty) {
-				inverse.fieldChange.newContent = { revert: revision };
+				assert(revision !== undefined, 0x478 /* Unable to revert to undefined revision */);
```

Comment:
```suggestion
				assert(revision !== undefined, "Unable to revert to undefined revision");
```

Reply:
Was this resolved by mistake?

Reply:
I was resolving stuff as I addressed it to keep track but hadn't pushed until just now.